docs/index.md

```diff
@@ -26,6 +26,8 @@ For more granular packages we recommend reading the documentation.
     - Authentication
         - [Shared access key authentication](features/security/auth/shared-access-key)
         - [Certificate authentication](features/security/auth/certificate)
+- **Correlation**
+    - [Correlate between request/responses](feature/correlation)
```

Comment:
```suggestion
    - [Correlate between request/responses](features/correlation)
```

Maybe we should be more precise and elaborate what we correlate?
`Provide request and/or transaction correlation ids`?

Reply:
👍